### DISCUSSION TITLE
About dataset

### DISCUSSION THREAD

**Discussion by NeethuGeorge-1** [bharathikannan1311/AI-for-Medicine-Specialization-deeplearning.ai#4] [Q&A] [answered]
Hi Bharathikannan,
        How to obtain the reduced dataset of ChestX-ray8 dataset(875-Train, 109 - Validation, 420 - Test) used in the week1 exercise of AI for Medical Diagnosis course?

**Comment by nbharathik** [comment, marked as answer]
Are you asking how to download those datasets to your local machine?

Refer coursera help center. Here is the link
https://learner.coursera.help/hc/en-us/articles/360004990332-Download-Jupyter-Workspace-files
I have downloaded all the files by this method only. It's easy to download all the files at once.  

Since datasets are large they are not uploaded here. Hope this helps.

**Comment by NeethuGeorge-1** [comment]
Thank you



With Regrads

On Tue, Jun 29, 2021 at 3:00 PM Bharathi Kannan ***@***.***>
wrote:

> Are you asking how to download those datasets to your local machine?
>
> Refer coursera help center. Here is the link
>
> https://learner.coursera.help/hc/en-us/articles/360004990332-Download-Jupyter-Workspace-files
> I have downloaded all the files by this method only. It's easy to download
> all the files at once.
>
> Since datasets are large they are not uploaded here. Hope this helps.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bharathikannann/AI-for-Medicine-Specialization-deeplearning.ai/discussions/4#discussioncomment-937254>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOJNDJ3EP7O3YZWEJBJQTA3TVGHDJANCNFSM44GKHUSA>
> .
>


-- 
Neethu George
Asst. Professor
Toc H Institute of Science & Technology
Arakunnam
